src/Pages/StatusPage/Status/Components/MonitorsList/index.jsx

```
@@ -8,7 +8,7 @@ import { StatusLabel } from "../../../../../Components/Label";
 import { useTheme } from "@mui/material/styles";
 import useUtils from "../../../../Uptime/Monitors/Hooks/useUtils";
 import PropTypes from "prop-types";
-const MonitorsList = ({ monitors = [] }) => {
+const MonitorsList = ({ isLoading = false, shouldRender = true, monitors = [] }) => {
```

ESLint

[error] 'isLoading' is missing in props validation

(react/prop-types)